spec/models/game_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Game, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
This same thing happened to me! Make sure to run the tests with bundle exec rspec and make sure they are all still passing. In this case it looks like we can either just delete this line or add a passing test.

Reply:
Do we have a preference if we'd like test or prefer deleting? Looks like with User/Devise we deleted? Thanks for feedback!

Reply:
I did go ahead and delete mine. I think that would be fine to do here until we really need any testing.

Reply:
Good Morning, changes should be completed. Let me know if I'm missing anything. Thanks again

Reply:
Good morning! I believe your branch was forked from a non-master branch. Running the current tests fail as it appears the databases were created on top of one of the unfinished angular branches. We do not want to currently merge this into the master branch

Reply:
I totally missed this the first time, sorry. Serves me right for not actually pull the branch and running the tests. I just knew these two pending ones wouldn't work.